source/unified-test-format/unified-test-format.rst

```diff
@@ -78,6 +78,10 @@ Iterable
   spec uses this as the return value for ``find`` and permit API flexibility
   rather than stipulate that a cursor object be returned directly.
 
+Server Compatibility
+--------------------
+
+The unified test format is intended to only run against enterprise servers.
```

Comment:
Consider adding: `with enableTestCommand=true` (or whatever the syntax is)

Reply:
Is the suggestion to clarify which tests / syntax doesn't work against community servers?

Reply:
Yes, it seems similar to the use case presented in DRIVERS-2698: if you try to run the tests without this they will fail (and in great numbers!).